nix/cloud/actions.nix

```diff
@@ -0,0 +1,23 @@
+{
+  cell,
+  inputs,
+}: {
```

Comment:
Why is this in a separate cell than the `pipelines.nix`?

Reply:
This is the naming convention we currently use for std.

The `automation` cell contains scripts and tasks for local execution.
The `cloud` cell contains only things that run on some hosted platform (like cicero).

Tullia tasks are valid standalone as they can be run locally without cicero, so they belong in the `automation` cell.
Cicero actions can only be run on cicero (and ours happen to run tullia tasks), so they belong in the `cloud` cell.